src/Avalonia.Base/Visual.cs

```diff
@@ -329,6 +329,7 @@ protected internal IAvaloniaList<Visual> VisualChildren
         /// <summary>
         /// Gets the control's parent visual.
         /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("AvaloniaProperty", "AVP1032", Justification = "GetVisualParent extension method is supposed to be used instead.")]
```

Comment:
Hmm this actually begs the question - why is `VisualParentProperty` public if this is internal. We should probably either make this public or make `VisualParentProperty` internal. Something for another PR though.